pkgs/top-level/all-packages.nix

```diff
@@ -540,6 +540,8 @@ with pkgs;
 
   comedilib = callPackage ../development/libraries/comedilib {  };
 
+  comfyui = callPackage ../applications/misc/comfyui { };
```

Comment:
You can try the new method: put the package in `pkgs/by-name/co/comfyui/package.nix`.
In this way you don't need to add it manually to all-packages.nix.